hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/scm/storage/ContainerProtocolCalls.java

```diff
@@ -228,35 +227,31 @@ public static XceiverClientReply putBlockAsync(
    * @param chunk information about chunk to read
    * @param blockID ID of the block
    * @param traceID container protocol call args
-   * @param excludeDns datamode to exclude while executing the command
```

Comment:
Why are we removing the exclude datanodes functionality from here?

Reply:
excludeDns is not used in putBlockAsync call.